### PR TITLE
Add GraphiQL link to topnav

### PR DESCRIPTION
**Pull request by sabondano**
## Motivation

* For BlockScout users to learn about our GraphQL API.

### Enhancements
* Editing `_topnav.html.eex` template to include link to `/graphiql`.
* Fixing `/api_docs/index.html.eex` template to use "BlockScout" instead
of "Explorer".

### CODE REVIEW

**Comment by sabondano** [comment]
Screen shot:

screen_shot_2018-10-31_at_4_32_21_pm

File: apps/block_scout_web/lib/block_scout_web/templates/layout/_topnav.html.eex [hunk 1]
```diff
@@ -56,13 +56,25 @@
...
+        <li class="nav-item dropdown">
+          <a href="#" role="button" id="navbarAPIsDropdown" class="nav-link topnav-nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
             <span class="nav-link-icon">
               <%= render BlockScoutWeb.IconsView, "_api_icon.html" %>
             </span>
-            <%= gettext("API") %>
-          <% end %>
+            <%= gettext("APIs") %>
+          </a>
+          <div class="dropdown-menu" aria-labeledby="navbarTransactionsDropdown">
```
Comment (KronicDeth) [inline apps/block_scout_web/lib/block_scout_web/templates/layout/_topnav.html.eex]: You may want to put `GraphQL` before `RPC`.  Users will likely pick the first entry on the dropdown when they are unsure.  By putting GraphQL first, we'll guide them to the API we want them to use.

Comment (acravenho) [reply apps/block_scout_web/lib/block_scout_web/templates/layout/_topnav.html.eex]: Very good point 👍

Comment (sabondano) [reply apps/block_scout_web/lib/block_scout_web/templates/layout/_topnav.html.eex]: Good call 👍

Comment (sabondano) [reply apps/block_scout_web/lib/block_scout_web/templates/layout/_topnav.html.eex]: ✅

**Pull request merged**